tests/test_distances.py

```diff
@@ -39,30 +40,17 @@ def test_haversine():
     assert distance == pytest.approx(distance_expected, rel=0.01)
 
 
-@pytest.mark.xfail(reason="API Key issues")
-def test_geocoding_airport_FRA():
+def test_geocoding_airport_FRA(mocker):
     """Test geocoding of airports using IATA code"""
```

Comment:
Would be helpful to mention in the docstring what should be tested exactly, like here I understand that it should be tested that an error is raised but I don't know what causes the error.